husc/pathutils.py

```diff
@@ -0,0 +1,34 @@
+import os
+from fnmatch import fnmatch
+
+def all_matching_files(path, glob='*.TIF', full=True, sort=True):
+    """Return list of files in directory with specified extension.
```

Comment:
Can probably be a bit more concise here: "Recurse all subdirectories of `path` and return all files matching `glob`."